src/draw_target.rs

```diff
@@ -368,6 +383,8 @@ pub(crate) struct ProgressDrawState {
     pub force_draw: bool,
     /// True if we should move the cursor up when possible instead of clearing lines.
     pub move_cursor: bool,
+    /// Controls how the multi progress is aligned if some its progress bars get removed, default is `Top`
```

Comment:
Probably just remove `its` here.

Reply:
Why is this duplicated between `ProgressDrawState` and `MultiProgressState`? Shouldn't one of them be enough?

Reply:
I see no easy way to pass the flag from `MultiProgressState` to `ProgressDrawTarget::apply_draw_state` function. So I used the similar pattern, that already exists for `move_cursor` flag.

Will try to find the better way.

Reply:
I'm fine with leaving it for now.